admin-dev/themes/new-theme/js/components/grid.js

```diff
@@ -51,6 +53,7 @@ export default class Grid {
     this._handleCommonGridActions();
     this._handleSortingGrid();
     this._enableDatePickers();
+    this._enableToggleForBooleans();
```

Comment:
it keeps getting bigger and bigger. what if we could do something like this:

```javascript
// column-toggle-handler.js

class ColumnToggleHandler() {
    run(event, grid) {
        // do stuff
    }
}

// some/specific/page/script.js

const grid = new Grid('#my-grid');

// you just add handlers that you actually need
// it can be the same for column, grid actions & etc.
// maybe fire some event so modules/other script can register their own handlers for the grid
grid.addColumnHandler(new ColumnToggleHandler());

grid.init();
```

@mickaelandrieu what do you think?

Reply:
yes, but in a specific Pull request pls :)

Can you think about a system when module developers can add more handlers without touch grid.js file, please? Create the issue on `prestafony-project`, you have already work to do today, isn't it?